packages/build/src/core/feature_flags.js

```diff
@@ -15,6 +17,8 @@ const getFeatureFlag = function (name) {
 }
 
 // Default values for feature flags
-const DEFAULT_FEATURE_FLAGS = {}
+const DEFAULT_FEATURE_FLAGS = {
+  zisiHandlerV2: env.NETLIFY_EXPERIMENTAL_FUNCTION_HANDLER_V2 === 'true',
```

Comment:
Note: LaunchDarkly flags should come from free with this PR, we just need to create the flag in the LaunchDarkly UI.